test/server/connection_handler_test.cc

```diff
@@ -149,7 +149,9 @@ class ConnectionHandlerTest : public testing::Test, protected Logger::Loggable<L
     void setDirection(envoy::config::core::v3::TrafficDirection direction) {
       direction_ = direction;
     }
-    Network::ConnectionBalancer& connectionBalancer() override { return *connection_balancer_; }
+    Network::ConnectionBalancer& connectionBalancer(const Network::Address::Instance&) override {
```

Comment:
Is there a test somewhere in this PR that actually tests multiple addresses and multiple balancers?

Reply:
I don't have a test for multiple addresses yet. I'm thinking to add integration test in the end
https://github.com/envoyproxy/envoy/blob/17d7d7908507888617824d73bc5904a3d79d1d17/test/integration/listener_lds_integration_test.cc#L1055

I should be able test multiple addresses for ConnectionHandler and ActiveTcpListener. ListenerImpl can't be tested now, since it isn't full support multiple addresses yet.

Reply:
A multiple address test added for connection handler https://github.com/envoyproxy/envoy/pull/21774/files#diff-3df0a7bd0288faa35b611e2bc11ea94713d9ca617ad16dfab81386c814f5e1eeR701